compiler/rustc_codegen_ssa/src/mir/mod.rs

```diff
@@ -226,6 +229,14 @@ pub fn codegen_mir<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>>(
             let layout = start_bx.layout_of(fx.monomorphize(decl.ty));
             assert!(!layout.ty.has_erasable_regions());
 
+            if layout.size.bytes() >= MIN_DANGEROUS_SIZE {
+                let size_str = || {
```

Comment:
Does this need to be a closure for some reason?

Reply:
No!